index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Nelson Mandela</title>
+</head>
+
+<body>
+    <div class="container">
+        <main>
+            <div id="img-area">
+                <h1>Nelson Mandela</h1>
+                <div class="img-div">
+                    <img class="img" src="./img/Nelson-Mandela-PNG-Images-HD.png" id="A photo of Nelson Mandela">
+                </div>
+                <small>
+                    <strong>Anti-apartheid leader Nelson Mandela arriving to address a mass rally, a few days after his release
+                        from jail, on Feb. 25 1990, in Bloemfontein, South Africa.</strong>
+                </small>
+            </div>
```

Comment:
I think it's better to use the header tag here instead of the div tag, it's best practice to use more semantic tags.